gdx/src/com/badlogic/gdx/graphics/g3d/environment/AmbientCubemap.java

```diff
@@ -141,10 +140,10 @@ public AmbientCubemap add (final Color color, final Vector3 point, final Vector3
 
 	@Override
 	public String toString () {
-		String result = "";
+		StringBuilder result = new StringBuilder();
 		for (int i = 0; i < data.length; i += 3) {
-			result += Float.toString(data[i]) + ", " + Float.toString(data[i + 1]) + ", " + Float.toString(data[i + 2]) + "\n";
+			result.append(Float.toString(data[i])).append(", ").append(Float.toString(data[i + 1])).append(", ").append(Float.toString(data[i + 2])).append("\n");

```

Comment:
Using `StringBuilder.append(data[i])` without `Float.toString()` should be even more efficient.

Reply:
[fix](https://github.com/libgdx/libgdx/pull/5670/commits/bbc13d702149f51a3ae2f1cc1179e47baecd4d89)